cpg_workflows/scripts/vep_json_to_ht.py

```diff
@@ -201,14 +201,11 @@ def vep_json_to_ht(vep_result_paths: list[str], out_path: str):
 
     ht = hl.import_table(paths=vep_result_paths, no_header=True, types={'f0': json_schema})
     ht = ht.transmute(vep=ht.f0)
-    # Can't use ht.vep.start for start because it can be modified by VEP (e.g. it
-    # happens for indels). So instead parsing POS from the original VCF line stored
-    # as ht.vep.input field.
-    original_vcf_line = ht.vep.input
-    start = hl.parse_int(original_vcf_line.split('\t')[1])
-    chrom = ht.vep.seq_region_name
-    ht = ht.annotate(locus=hl.locus(chrom, start))
     # we're using split multiallelics, so we need to compound key on chr/pos/ref/alt
+    ht = ht.annotate(
+        locus=hl.locus(ht.vep.seq_region_name, hl.parse_int(ht.vep.input.split('\t')[1])),
+        alleles=ht.vep.allele_string.split('/'),
```

Comment:
Is it easy to make this split on '/' OR '|'

Given our push to LR we are going to start seeing phased VCFs more and more so it would be good to build that into our muscle memory

Reply:
A split on one of two different separators isn't explicitly available as a hail method, but we can do a replace for `|` -> `/`, then split on the latter 🤔

Reply:
Not a bit deal for this PR in particular - just one of those footguns that makes me nervous about basic string parsing a VCF.

Reply:
I'll bookmark this and have a look for some phased calls in the final MT. I think this might be fine as this is the Alleles string, but the phasing is in the GT field. I would expect that even for `0|1` the alleles being passed to VEP are still in `A/T` format, with the option of being `A/C/T` for multiallelics